### PR TITLE
pass scm and scm_packages attributes to resource

### PR DESCRIPTION
**Pull request by evandam**
# Description

Pass extra attributes to resource in default recipe.

## Issues Resolved

Fixes https://github.com/sous-chefs/golang/issues/88

## Check List

- [ ] All tests pass. See TESTING.md for details.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable.

### CODE REVIEW

**Review by jeffbyrnes — DISMISSED** [review]
Looks good, though you’ll need to update the CHANGELOG before this can merge.

**Comment by evandam** [comment]
Thanks @jeffbyrnes - updated the changelog assuming this is a patch.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 This file is used to list changes made in each version of golang.
 
+## 4.1.1
```
Comment (ramereth) [inline CHANGELOG.md:5]: We need this so our auto-release will work.
```suggestion
## Unreleased
```

**Comment by evandam** [comment]
Hey @ramereth, sorry for the delay, must've missed a notification. Just updated the changelog as you recommended.

**Review by ramereth — APPROVED** [review]

**Comment by kitchen-porter** [comment]
Released as: [4.1.1](https://github.com/sous-chefs/golang/releases/tag/4.1.1)

**Pull request merged**